docs/stdlib/numbers.rst

```diff
@@ -207,39 +210,48 @@ from :eql:type:`str` and :eql:type:`json`.
 
     :index: numeric bigint
 
-    Arbitrary precision integer.
+    Represents an arbitrary precision integer.
 
-    The EdgeDB philosophy is that using bigint type should be an
-    explicit opt-in, but once used, the values should not be
-    accidentally cast into a numeric type with less precision.
+    In EdgeDB, the philosophy of :eql:type:`bigint` is that using this type
+    should be an explicit opt-in and not implicit. Once used, these values
+    should not be accidentally casted to a different numerical type that gives
+    less precision.
 
-    In accordance with this :ref:`the mathematical functions
-    <ref_std_math>` are designed to keep the separation
-    between bigint values and the rest of the numeric types.
+    In addition with this, :ref:`our mathematical functions <ref_std_math>`
+    are designed to keep separation between big integer values and the
+    rest of our numerical types.
```

Comment:
```suggestion
    In keeping with this philosophy,
    :ref:`our mathematical functions <ref_std_math>`
    are designed to maintain separation between big integer values and the rest
    of our numeric types.
```

- "In addition with" changes the meaning here The way our functions work _maintains_ the philosophy rather than being additive to it.
- Reverted "numerical" to "numeric" since it's not adding any clarification here